src/applications/static-pages/shared/ExpandableOperatingStatus.jsx

```diff
@@ -0,0 +1,86 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import recordEvent from 'platform/monitoring/record-event';
+
+const ExpandableOperatingStatus = props => {
+  const [open, setOpen] = useState(false);
+
+  const handleOnclick = e => {
+    if (!props.extraInfo) return;
+    if (e) e.target.classList.toggle('active');
+    setOpen(!open);
+    if (!open) {
+      recordEvent({
+        event: 'int-accordion-expand',
+        'operating-status-heading': props.statusLabel,
+      });
+    } else {
+      recordEvent({
+        event: 'int-accordion-collapse',
+        'operating-status-heading': props.statusLabel,
+      });
+    }
+  };
+
+  const iconIndicator = extraInfo => {
+    if (extraInfo && open) return 'fa-angle-down open';
+    if (extraInfo && !open) return ' fa-angle-down close';
+    return null;
+  };
+
+  const outputLinks = data => {
```

Comment:
Can we add a unit test for this function?

Reply:
not here, let's add it for the filter, it's the same function